data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+			choice
+				`	"They attacked me first."`
+				`	"I wanted to get their attention."`
+					goto attention
+				`	"Violence happens all the time in human space, but the Quarg don't seem interested in it."`
+					goto violence
+
+			`	The Quarg once again pauses for a long moment, staring at you. "Based on some principles, I may see reason for them to be, inclined, to do such. However, to do so in this case would not line with their elder's interest, nor would any, fabrications." The Quarg blinks at you, then ponders for a moment.`
+				goto cases
+
+			label attention
+			`	The Quarg blinks. "And you found a suitable solution to your, problem, by instigating unprovoked hostilities?" The Quarg then continues to stare at you in what you presume to be a judgemental manner for an awkward amount of time, then ponders to itself.`
+				goto cases
+
+			label violence
+			`	The Quarg takes a long moment to ponder your statement, as if it wasn't expecting such a response. "Indeed, it is known that conflict is commonly natural amongst you quarrelsome humans. But this matter is of another sort." The Quarg then thinks to itself again for a little longer.`
+
+
+			label cases
+			`	"Imagine, if you will, during the substantial conflicts the humans call the Turf Wars, a hostile alien came to human space. We Quarg woefully witnessed that hazardous conflict - it was a time of great waste, unfortunate peril, and unsavory distrust among humanity. The Gegno are in a vulnerable state notably comparable to such a time, and introduction of even more hostilities would not befit the case, especially of those from the outside. We have been contacted by numerous members of high Gegno standing and wish for it to be matter only for further strict cautionaries, but regrettably, the Gegno believe it a matter beyond intricate attentiveness." The Quarg leaves its chair and motions you to do the same.`
+			choice
+				`	(Stand up.)`
+
+			`	"You are recommended for departure from Gegno space as of this decisive moment. Meanwhile, we shall attempt a most careful negotiation with their hierarchy to come in terms of an acceptable toleration of storts. Should we find commonality, you will receive communication from one of the title 'Adrauni', as has been bestowed upon me."`
+			`	As the rest of the Quarg escort you out of the room, the Quarg named 'Adrauni' addresses you one last time. "Do avoid any further, violent tendencies, captain." If the Gegno are willing to go as far as to contact the Quarg, then this is truly an important matter to them. For the time being, it's probably best to listen to the Quarg's warnings this time around and leave Gegno space until some kind of resolution is found, if at all.`
+				decline
+
+	on decline
+		event "gegno: quarg talks" 11
```

Comment:
Might be cool to have this event take longer to fire if the player came into this with negative rep.
Could implement it as `on accept` for the neutral option and `on decline` for the negative one